accelerator/cli_info.h

```diff
@@ -95,7 +95,7 @@ static struct ta_cli_argument_s {
      "The period for checking IOTA full node host connection status"},
     {"no-gtta", no_argument, NULL, NO_GTTA, "Disable getTransactionToConfirm (gTTA) when sending transaction"},
     {"buffer_list", required_argument, NULL, BUFFER_LIST, "Set the value of `buffer_list_name`"},
-    {"done_list", required_argument, NULL, DONE_LIST, "Set the value of `done_list_name`"},
+    {"complete_list", required_argument, NULL, COMPLETE_LIST, "Set the value of `complete_list_name`"},
```

Comment:
Both `buffer_list` and `complete_list` should come with human-readable descriptions.